src/main/java/uk/gov/hmcts/reform/civil/service/DashboardNotificationsParamsMapper.java

```diff
@@ -50,6 +53,9 @@ public HashMap<String, Object> mapCaseDataToParams(CaseData caseData) {
         } else {
             params.put("djDefendantNotificationMessage", "<u>make an application to set aside (remove) or vary the judgment</u>");
         }
+        if (caseData.getJoJudgmentRecordReason() != null && caseData.getJoJudgmentRecordReason().equals(JudgmentRecordedReason.DETERMINATION_OF_MEANS)) {
+            params.put("paymentFrecuencyMessage", getPaymentFrecuencyMessage(caseData).toString());
+        }
```

Comment:
What happend if its not Pay_By_Installments , does it add a empty string in map?